drivers/hopwa_caper/app/models/hopwa_caper/filters/hopwa_caper_filter.rb

```diff
@@ -0,0 +1,13 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HopwaCaper::Filters
+  class HopwaCaperFilter < ::Filters::HudFilterBase
+    def funder_ids
+      HudUtility2024.funder_components.fetch('HUD: HOPWA')
+    end
+  end
```

Comment:
This is probably fine, but it is a pain point we've fought to avoid in the past.  Rather than extending `Filters::HudFilterBase` you can just set the `funder_ids` attribute.  Again, maybe it's fine, but we had an explosion of filter classes that ended up being really hard to maintain so we've tried to keep it just to the base filter and HUD base filter... somewhat unsuccessfully.

Reply:
got it, I can change that behavior